labs/09_gcs/README.md

```diff
@@ -0,0 +1,107 @@
+# (Bonus) Lab Week 09: Google Cloud Storage (GCS)
+
+This is just a quick example lab to show [Google Cloud Storage](https://cloud.google.com/storage/docs/introduction), what it can do and how it can be used.
+
+Google Cloud Storage (GCS) is a highly durable and highly available object storage service offered by Google Cloud.
+
+It is usually used to store and access any amount of data from anywhere in the world.
+
+It is included in this course as it is a convenient way to **stage data so it is available by different services in the Cloud**.
+
+:point_right: For example, you built a **dashboard** that should access **data** produced by another service (or model), then you can use GCS for that. Note that there are better suited services such as [BigQuery](https://cloud.google.com/bigquery/docs) for that but let's keep it simple here.
+
+## 0. Setup
+
+Make sure to have a GCP project you can use
+1. Visit the [Google Cloud Console](https://console.cloud.google.com/) and either select a project or create a new one.
+2. :sunglasses: Note that you can give access to other users to your GCP project through the [IAM page](https://console.cloud.google.com/iam-admin/) - so multiple project can work on one project **with the same billing account**
+3. Make sure to have a [billing account](https://console.cloud.google.com/billing/) (remember that you can request credits to the teaching staff!)
+
+Enable the Cloud Storage API:
+1. Navigate to the [Cloud storage tab](https://console.cloud.google.com/storage)
+2. If requested Enable the API
+
+And we will then **create a bucket** in which we will store the files for this lab.
+
+You can do so from the UI directly.
```

Comment:
What about all the possible options that can be chosen (e.g. region : single region, etc.) ?

Reply:
Will make sure to voice that different regions and availability can be picked